<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QDialog" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controller</string>
  </property>
  <widget class="QDialogButtonBox" name="closeButton">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_address">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>201</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CONNECT!!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_connections">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connections:</string>
   </property>
  </widget>
  <widget class="QLabel" name="highlightedConnection">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Awaiting commands</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pingButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EXECUTE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_command">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_tweet">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>270</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tweetButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TWEET!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_dos">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dosButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DoS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>accepted()</signal>
   <receiver>Controller</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>rejected()</signal>
   <receiver>Controller</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
